MetamodelAlgemeen.md

```diff
@@ -1518,6 +1518,8 @@ betreft, zoals een `CharacterString`.
 
 *Toelichting*
 
+Een abstract objecttype moet altijd de generalisatie zijn van één of meerdere objecttypes die niet abstract zijn.
```

Comment:
Dit hoeft **zeker** niet altijd het geval te zijn. Het komt juist heel vaak voor dat je meerdere lagen van abstracte klasses heb voordat je bij een concrete klasse komt. Bijvoorbeeld: Geo-Object -> RegistratieveRuimte -> BestuurlijkGebied -> Gemeentegebied.

Reply:
dank je wel. Helemaal gelijk.

Reply:
is nu verwijderd